apps/store/src/components/CartPage/CartPage.tsx

```diff
@@ -76,7 +76,7 @@ const EmptyState = ({ children, prevURL }: EmptyStateProps) => {
         <Space y={2}>
           <Space y={1}>
             <CenteredText>¯\_(ツ)_/¯</CenteredText>
-            <CenteredText>{t('CART_EMPTY_SUMMARY')}</CenteredText>
+            <CenteredTextSecondary>{t('CART_EMPTY_SUMMARY')}</CenteredTextSecondary>
```

Comment:
We can use props for `color` and `alignment` int this case  😃
```suggestion
            <Text color="textSecondary" align="center">{t('CART_EMPTY_SUMMARY')}</Text>
```

Reply:
haha sorry, didn't see @robinandeer comment 😁

Reply:
Tack! did not know it was a thing 🧠